hls4ml/templates/vivado/nnet_utils/nnet_conv1d_latency.h

```diff
@@ -84,5 +84,85 @@ void conv_1d_latency_cl(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan],
     }
 }
 
+template <class data_T, class res_T, typename CONFIG_T>
+void pointwise_conv_1d_latency_cl(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan / CONFIG_T::reuse_factor],
```

Comment:
The files in Vitis/Vivado are mostly identical (apart from the very recent change in the final loop), perhaps we could remove the Vitis one (first we follow up with the changes)

Reply:
Do you mean both `nnet_conv1d.h` and `nnet_conv1d_latency.h`?

There are a few differences like the use of inline region, etc. Do you mean to test if just using the vivado versions works in vitis now?